.github/workflows/release-starter.yml

```diff
@@ -0,0 +1,132 @@
+name: Release starter
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The new version to release (`v2.16.0`)
+        required: true
+        type: string
+      pre-release:
+        description: Is this a pre-release ?
+        required: true
+        default: true
+        type: boolean
+
+# We set `concurrency` to prevent having this workflow being run multiple time for the same version
+concurrency:
+  group: release-starter-${{ inputs.version }}
+  cancel-in-progress: true
+
+jobs:
+  release-starter:
+    runs-on: ubuntu-22.04
+    name: Release starter 🚛
+    steps:
+      - name: Install python-semver-3.0.0
+        run: python3 -m pip install 'semver~=3.0.0'
```

Comment:
we already have version parsing code in `releaser.py`, why not adding a new command to this script to utpout the file you need:
```shell
$ ./misc/releaser.py parse-version v2.16.0-rc1
full=2.16.0-rc1
major=2
minor=16
patch=0
prerelease=rc1
build=
```

hence the code here would be:
```shell
$ ./misc/releaser.py parse-version ${{ inputs.version }} > os.environ["GITHUB_OUTPUT"]
```

Reply:
done